index.html

```diff
@@ -52,13 +55,13 @@ <h3>Generic Use-Case</h3>
         <h3>Practical Use-Case</h3>
         <p>
           Consider a combo-application consisting of two major parts - a video-conferencing
-          application and a productivity-suite application co-existing in a [=browsing context=].
-          Assume the video-conferencing uses existing/upcoming APIs such as
+          application and a productivity-suite application co-existing in a shared [=top-level
+          browsing context=]. Assume the video-conferencing uses existing/upcoming APIs such as
           {{MediaDevices/getDisplayMedia()}} and/or <code>getViewportMedia</code> and captures the
-          entire [=browsing context=]. Now it needs to crop away everything other than a particular
-          section of the productivity-suite. It needs to crop away its own video-conferencing
-          content, any speaker notes and other private and/or irrelevant content in the
-          productivity-suite, before transmitting the resulting cropped video remotely.
+          entire [=top-level browsing context=]. Now it needs to crop away everything other than a
```

Comment:
_"captures the entire [=top-level browsing context=]"_ seems ambiguous. We're not capturing history (bfcache) documents it contains, but we are capturing active documents in nested browsing contexts. How about:
```suggestion
          entire [=top-level browsing context=]'s <a data-cite="HTML#viewport">viewport</a>. Now it needs to crop away everything other than a
```

Reply:
Same comment about what display-capture deals with. E.g. we're also not capturing hidden state, such as the values of variables, obscured content, etc. I am not very opposed to clarifying "viewport" on its own, but I am concerned we're hitting the [heap paradox](https://en.wikipedia.org/wiki/Sorites_paradox) by incrementally making the example ever more precise and less readable.

Reply:
Yes, as I mentioned [above](https://github.com/w3c/mediacapture-region/pull/41#discussion_r840820367), I don't think normative links belong in examples, and we should rewrite this with layperson terms and mark  [§ 3. Use Cases](https://w3c.github.io/mediacapture-region/#use-cases) as non-normative. That is my preferred resolution. Or we can add the word "viewport" to clarify that "entire" implies to cropping and the rendered form of the viewport, and not the browsing context.

I like to give options.

Reply:
Let's go with "tab" then. And I've added the non-normative marking to [§ 3. Use Cases](https://w3c.github.io/mediacapture-region/#use-cases).